Tribler/Core/Modules/restapi/display_endpoint.py

```diff
@@ -81,7 +86,7 @@ def render_GET(self, request):
 
             .. sourcecode:: none
 
-                curl -X GET http://localhost:8085/display?focus_node=xyz
+                curl -X GET http://localhost:8085/display?dataset=static&focus_node=xyz
```

Comment:
It is not ignored, the only influence the dataset argument has on the focus_node argument is that when dummy data is used, 'self' should default to "0" instead of your own public key